rep-0145.rst

```diff
@@ -94,20 +94,22 @@ Topics
 The following topics are expected to be common to many devices - an IMU device driver is expected to publish at least one. Note that some of these topics may be also published by support libraries, rather than the base driver implementation. All message types below are supplemented with a std_msgs/Header, containing time and coordinate frame information.
 
 
-* `imu/data_raw` (sensor_msgs/Imu)
+* `imu/angular_velocity` (sensor_msgs/AngularVelocity)
 
   - Sensor output grouping accelerometer (`linear_acceleration`) and gyroscope (`angular_velocity`) data.
 
-* `imu/data` (sensor_msgs/Imu)
+* `imu/attitude` (sensor_msgs/Attitude)
 
-  - Same as `imu/data_raw`, with an included quaternion orientation estimate (`orientation`).
+  - Sensor output containing information about the attitude
 
-* `imu/mag` (sensor_msgs/MagneticField)
+* `imu/linear_acceleration` (sensor_msgs/LinearAccleration)
```

Comment:
```suggestion
* `imu/linear_acceleration` (sensor_msgs/LinearAcceleration)
```

Reply:
Maybe move `linear_acceleration` up one bullet point, so it's right behind `angular_velocity`? Makes more sense to group these two together, because those are usually the raw sensor data, and `attitude` is computed from those, so it shouldn't be between them. We could also move `attitude` right to the end (after `angular_velocity`, `linear_acceleration` and `mag`).